src/components/VoteActions/VoteActions.tsx

```diff
@@ -8,20 +8,16 @@ import VoteResults from 'components/VoteResults';
 import { getWinningText } from 'utilities/helpers';
 import { useVoteEndContext } from 'common/VoteEndContext';
 import { useFipDataContext } from 'common/FipDataContext';
+import Loading from 'common/Loading';
 import VotePicker from './VotePicker';
+import useVoteResults from '../../hooks/useVoteResults';
```

Comment:
Nit: Looks like you have this set up so you could just use the path `hooks/useVoteResults`

Reply:
@lmao12 yeah, but for some reason TS was breaking so I just gave up and used the absolute path, haha